api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_highly_variable_genes.py

```diff
@@ -322,21 +319,20 @@ def get_highly_variable_genes(
     organism: str,
     measurement_name: str = "RNA",
     X_name: str = "raw",
-    obs_value_filter: Optional[str] = None,
-    obs_coords: Optional[SparseDFCoord] = None,
-    var_value_filter: Optional[str] = None,
-    var_coords: Optional[SparseDFCoord] = None,
+    obs_value_filter: str | None = None,
```

Comment:
AFAIK, the `|` type syntax was added in Python 3.10, but this package supports back to Python 3.8.  This seems likely to fail on Python <3.10, right?